src/NovelRT/Timing/StepTimer.cpp

```diff
@@ -21,17 +19,21 @@ namespace NovelRT::Timing {
     _isFixedTimeStep(targetFrameRate != 0) {
   }
 
+  void StepTimer::initializeValues() {
+    _frequency = glfwGetTimerFrequency();
+    _lastCounter = glfwGetTimerValue();
+  }
+
   void StepTimer::resetElapsedTime() {
-    _lastCounter = glfwGetTime();
+    _lastCounter = glfwGetTimerValue();
     _secondCounter = 0;
     _remainingTicks = 0;
     _framesPerSecond = 0;
     _framesThisSecond = 0;
   }
 
   void StepTimer::tick(const std::vector<NovelUpdateSubscriber>& update) {
-
-    auto currentCounter = glfwGetTime();
+    auto currentCounter = glfwGetTimerValue();
```

Comment:
We were incorrectly getting the value in seconds, and not the raw tick count. (Thanks @tannergooding for explaining this one)